apps/frontend/src/libs/components/select/select.tsx

```diff
@@ -48,6 +48,34 @@ const Select = <TFieldValues extends FieldValues, TOptionValue>({
 		isLabelHidden && "visually-hidden",
 	);
 
+	const handleChange = useCallback(
```

Comment:
We need to return changes in select component back, it should be controlled using useForm

Reply:
Could you please explain a bit more what should be here? I added handleChange just so select would add only value to formData fields, not label value object.

Reply:
Oh, I see, I was thinking you added custom handlers. in this case we need to simplify this logic, because it's not very readable. Something like:
```typescript
	const findOptionByValue = useCallback(
		(value: TOptionValue): SelectOption<TOptionValue> | undefined => {
			return options.find((option) => option.value === value);
		},
		[options],
	);
	
	const selectedValue = useMemo(() => {
		if (isMulti) {
			return (field.value as TOptionValue[])
				.map(findOptionByValue)
				.filter(Boolean);
		}

		return findOptionByValue(field.value as TOptionValue);
	}, []);
```

And for handleChange, can't we use `let` with if instead of single multiline condition? or linter doesn't allow this?

Reply:
Simplified the logic. For handleChange linter allows only ternary expression.